src/scripts/components/PillsInput.jsx

```diff
@@ -76,11 +79,13 @@ PillsInput.propTypes = {
   initialInputValue: PropTypes.string.isRequired,
   pillType: PropTypes.string,
   pillCloseIconClassName: PropTypes.string,
+  inputProps: PropTypes.object,
```

Comment:
This one is probably a requirement no matter what so let's mark it as such.

Also let's add a shape to this object that corresponds to what the inputProps look like rather than an object.

E.G:
```
inputProps: PropTypes.shape({
    name: PropTypes.string.isRequired,
    format: PropTypes.bool,
    onChange: PropTypes.func,
  ...etc
  }),
```